sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala

```diff
@@ -92,6 +94,9 @@ private[sql] class JSONOptions(
   val timestampFormat: String =
     parameters.getOrElse("timestampFormat", "uuuu-MM-dd'T'HH:mm:ss.SSSXXX")
 
+  val intervalOutputStyle: IntervalStyle = parameters.get("intervalOutputStyle")
```

Comment:
can we add it in another PR? we need to update docs and pyspark side, which is non-trivial.

Reply:
OK, I will keep the interval in json format to be multi-units style